code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -273,12 +273,12 @@
 	if(!on || client)
 		return
 
-	if(!commissioned && can_salute)
-		for(var/mob/living/simple_animal/bot/B in get_hearers_in_view(5, get_turf(src)))
-			if(B.commissioned)
-				visible_message("<b>[src]</b> performs an elaborate salute for [B]!")
-				can_salute = FALSE
-				addtimer(VARSET_CALLBACK(src, can_salute, TRUE), salute_delay)
+	if(commissioned && can_get_salute)
+		can_get_salute = FALSE
+		addtimer(VARSET_CALLBACK(src, can_get_salute, TRUE), salute_delay)
```

Comment:
Could you use one of the `cooldowns.dm` systems for standardization?
You can either keep the a variable such as `salute_delay` to store the cooldown, or you can just use the timer cooldown and eschew that.

Reply:
Never used cooldowns.dm before. I think I did it right?

Reply:
This is good!